bin/cake

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env node --harmony
```

Comment:
bin/cake is "public". Several projects written in CoffeeScript use `cake` as their build tool. How will this change affect them? Is it safe to run with `--harmony`?

This change is for the tests, right? So I guess `npm run test-harmony` would do the trick just as well? If so, perhaps it makes sense to try to:

- Make `npm test` run `npm run test-harmony` instead.
- Make `cake test` error and suggest running `npm test` instead if it detects that `--harmony` wasn't provided.